### PR TITLE
Reorder description

### PR DESCRIPTION
**Pull request by YC-1412**

### CODE REVIEW

**Comment by bojum** [comment]
Won’t this create even more conflicts?

**Comment by YC-1412** [comment]
No, I didn't change anything except for reordering them.

File: DESCRIPTION [hunk 1]
```diff
@@ -5,32 +5,32 @@ Version: 0.0.1
...
-  wordcloud2,
-  dplyr,
+  reshape2,
+  Rmpfr,
   shiny,
-  devtools,
-  Rmpfr
+  stringr,
+  tidyverse,
+  tidyverse,
```
Comment (skyetim) [inline DESCRIPTION]: Remove duplicate

**Review by skyetim — CHANGES_REQUESTED** [review]

**Review by skyetim — APPROVED** [review]

**Comment by skyetim** [comment]
No longer applicable. Need to do it again after everything is merged.

**Pull request closed without merging**